var/spack/repos/builtin/packages/multicharge/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Multicharge(CMakePackage):
```

Comment:
This package also supports meson as build system, if used in dftd4 as dependency, it should be available via meson there as well.

Reply:
I added only the CMake builder, because this package did not seem to be required to build `dftd4` with `meson`. I can certainly add the `meson` builder.

Reply:
I added the `meson` build system for the `multicharge` package.